components/layout_2020/fragments.rs

```diff
@@ -34,7 +34,10 @@ pub(crate) enum Fragment {
 }
 
 #[derive(Serialize)]
-pub(crate) struct AbsoluteOrFixedPositionedFragment(pub HoistedFragmentId);
+pub(crate) struct AbsoluteOrFixedPositionedFragment {
+    pub position: ComputedPosition,
+    pub hoisted_fragment: ArcRefCell<Option<ArcRefCell<Fragment>>>,
```

Comment:
I think this could be `Arc<ArcSwapAny<Option<ArcRefCell<Fragment>>>>` but the complexity of [arc-swap](https://github.com/vorner/arc-swap) is non trivial so let’s stick with the double `ArcRefCell` for now. We can always reevaluate later.